clang/docs/LanguageExtensions.rst

```diff
@@ -1911,6 +1911,38 @@ A simplistic usage example as might be seen in standard C++ headers follows:
   // Emulate type trait for compatibility with other compilers.
   #endif
 
+
+.. _builtin_structured_binding_size-doc:
+
+__builtin_structured_binding_size (C++)
+---------------------------------------
+
+The ``__builtin_structured_binding_size(T)`` type trait returns
+the *structured binding size* ([dcl.struct.bind]) of type ``T``
+
+This is equivalent to the size of the pack ``p`` in ``auto&& [...p] = declval<T&>();``.
+If the argument cannot be decomposed (ie not a builtin array, builtin SIMD vector,
```

Comment:
I feel a bullet list would be better here. Also a positive form would read better, the following can be decomposed, otherwise it is not well-formed something like that.